src/app/layout.tsx

```diff
@@ -3,6 +3,7 @@ import { Poppins } from "next/font/google";
 import { Metadata } from "next";
 import ReduxProvider from "@/redux/ReduxProvider";
 import { Toaster } from "sonner";
+import AnimatedCursor from "react-animated-cursor";
```

Comment:
the library added in package and in here are different, why?

Reply:
As I raised the issue to add tilt effect and animated cursor but I found no cards where I could add the tilt effect, so just add the libraries but not used  it. It will probably be used once there are cards present in your site.

Reply:
Where did you install the animated cursor library?

Also to create those tilt card, can you use the github apis to fetch top 3 or top 5 contributor's profile to display?

Reply:
> Where did you install the animated cursor library?
> 
> Also to create those tilt card, can you use the github apis to fetch top 3 or top 5 contributor's profile to display?

In package.json , I need to lookout for that and will surely raise an issue regarding same.